packages/cli/openapi-parser/src/options.ts

```diff
@@ -9,11 +9,14 @@ export interface ParseOpenAPIOptions {
     useTitlesAsName: boolean;
     /* The audiences that the OpenAPI must be filtered down into */
     audiences: string[] | undefined;
+    /* Whether or not to make additional proeprties optional */
```

Comment:
```suggestion
    /* Whether or not to make additional properties optional */
```

Reply:
Could also make this _slightly_ more clear and say "... property _values_ optional".

Reply:
updated, thank you!